### ISSUE TITLE
Add a11y into essentials

### ISSUE THREAD

**Issue by WilliamCSA04** [storybookjs/storybook#12485]
Hi. I came here to request that [a11y addon](https://www.npmjs.com/package/@storybook/addon-a11y) to be added into [essentials](https://www.npmjs.com/package/@storybook/addon-essentials).

I think that front-end developers should care a lot about it even tho this is not so popular this is really important for some users and we need to be sure that those users can use our features just like everybody else. Also, I think this is a good way to incentivize more developers to care about it.
 
I gladly make a pull request doing it if everyone agrees that is important. I would love to contribute to that! :heart:

Thank you for your time!

**Comment by shilman** [comment]
I'm going to write up some requirements for addons to get merge into essentials. Once a11y satisfies those, we should be able to add it.
